### PR TITLE
UI tests for credit card checkout [TPWDCEE-5403]

### PR DESCRIPTION
**Pull request by pretzelhands**

### CODE REVIEW

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- wirecardpaymentgateway/controllers/front/payment.php  1
         

Complexity decreasing per file
==============================
+ wirecardpaymentgateway/views/js/creditcard.js  -1
         

Clones removed
==============
+ wirecardpaymentgateway/views/js/creditcard.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/prestashop-ee/prestashop-ee/pullRequest?prid=4416095&bid=14956609)

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- wirecardpaymentgateway/controllers/front/payment.php  1
         

Complexity decreasing per file
==============================
+ wirecardpaymentgateway/views/js/creditcard.js  -1
         

Clones removed
==============
+ wirecardpaymentgateway/views/js/creditcard.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/prestashop-ee/prestashop-ee/pullRequest?prid=4416095&bid=14956609)

**Comment by rinnhofer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- wirecardpaymentgateway/controllers/front/payment.php  1
         

Complexity decreasing per file
==============================
+ wirecardpaymentgateway/views/js/creditcard.js  -1
         

Clones removed
==============
+ wirecardpaymentgateway/views/js/creditcard.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/prestashop-ee/prestashop-ee/pullRequest?prid=4416095&bid=14956609)

File: wirecardpaymentgateway/tests/acceptance/4-CreditCard3DSPurchaseHappyPath.feature [hunk 1]
```diff
@@ -5,9 +5,10 @@ Feature: CreditCard3DSPurchaseHappyPath
 
   Background:
     Given I activate "creditcard" payment action "pay" in configuration
+    Given I activate one-click checkout in configuration
```
Comment (tomazpu) [inline wirecardpaymentgateway/tests/acceptance/4-CreditCard3DSPurchaseHappyPath.feature]: This is wrong usually you use given something and something and something... not Two Given states

File: wirecardpaymentgateway/tests/acceptance/5-CreditCard3DSOneClickHappyPath.feature [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Feature: CreditCard3DSOneClickHappyPath
+  As a registered user
+  I want to make a one-click checkout with a Credit Card 3DS
+  And to see that transaction was successful
+
+  Background:
+    Given I activate "creditcard" payment action "pay" in configuration
```
Comment (tomazpu) [inline wirecardpaymentgateway/tests/acceptance/5-CreditCard3DSOneClickHappyPath.feature]: Why is here no given i activate one-click?

**Pull request merged**